### DISCUSSION TITLE
A hitchhiker's guide to implementing Mythic difficulty

### DISCUSSION THREAD

**Discussion by jyossarian** [chromiecraft/chromiecraft#7340] [Suggestions]
For your convenience, a quick guide on How to Implement Mythic Difficulty for BT release. Follow at your own risk (of an immense success, it's so simple it has to work!)

The premise is simple:

**+1 item drop per boss on Mythic difficulty**

That's basically it.

Why is it better?

You don't lock "bad luck protection" behind a Mythic difficulty. On a "mostly casual" server such as CC, do you really want to lock this feature behind a custom difficulty? It creates unnecessary pressure on "casual" guilds to do Mythic to secure BIS gear, excluding the exact player base you want to entice by introducing separate raid difficulties.

Bosses dropping 4 instead of 3 items on Mythic is more than enough incentive for guilds to try the harder difficulty. 

"Casual" guilds will have more fun trying to push each extra boss, without feeling like they're missing out on major rewards.

Implementation is super easy.

Conclusion:

PLEASE! Just add +1 loot drop to every boss on Mythic and call it a day. 

Everyone will love the challenge of an extra difficulty, with some extra loot and 0 pressure; because it's not guaranteed gear via the "bad luck protection".

**Comment by vvranceanu** [comment]
This, together with the progression system making tier tokens drop 2 instead of 3, makes so much sense. 3 items, 4, eh... whatever. Mythic? Its just bad luck protection.

But we have progression module!

Too good.

**Comment by Never6** [comment]
this is fine, as I mainly wanted mythics to return for the sole reason of getting 8month elusive items

**Comment by AcetylGer** [comment]
Not a fan of this suggestion as it creates a lot more void shards.
Could aswell just do 3 items on mythic and -1 item per boss on normal.

I think the whole allure of mythic is getting the guaranteed items **after** clearing mythic. 
Currently you need to clear the whole 9/9 on mythic to get any BT items, as stated by Nyeriah . Points accumulate during progression.
If you just do +1 item per boss people get bonus items during each week instead of having to beat the raid on mythic and being rewarded with an item of their choice after 4/5 weeks.
